src/Sarif.FunctionalTests/v2/SpecExamples/Comprehensive.sarif

```diff
@@ -38,10 +38,18 @@
           "branch": "master",
           "tag": "beta1",
           "revisionId": "abcdabcdabcdabcdabcdabcdabcdabcdabcdabcd",
-          "timestamp": "2016-07-16T00:00:00Z"
+          "timestamp": "2016-07-16T00:00:00Z",
+          "mappedTo": {
+            "uri": "/example",
+            "uriBaseId": "SRCROOT"
+          }
         }
       ],
-      "originalUriBaseIds": { "TOOLS_ROOT": "file:///bin/tools/" },
+      "originalUriBaseIds": {
+        "TOOLS_ROOT": {
```

Comment:
>TOOLS_ROOT [](start = 9, length = 10)

I noticed a couple of other comprehensive sample bugs while I was in here. I filed an issue to bring it up to date, and another issue to add a unit test to validate it.

Reply:
This code is implicit testing for our transformation logic. Can you please revert it? I've mentioned it previously but nearly all checked in SARIF is pre-release v2. We use this to leverage that content to test the transformation logic. Once our design is hardened, we will complete a massive push to upgrade all the existing content. 

---
In reply to: [244754797](https://github.com/Microsoft/sarif-sdk/pull/1198#discussion_r244754797) [](ancestors = 244754797)